sql/codegen.go

```diff
@@ -271,13 +290,13 @@ classifier.compile(optimizer=classifier.default_optimizer(),
 	loss=classifier.default_loss(),
 	metrics=["accuracy"])
 classifier.fit(train_input_fn(X, Y, BATCHSIZE),
-	epochs=classifier.default_training_epochs(),
-	steps_per_epoch=STEP, verbose=0)
+	epochs=EPOCHS if EPOCHS != classifier.default_training_epochs() else classifier.default_training_epochs(),
```

Comment:
In the current configuration, I think `classifier.default_training_epochs()` will never be used...  Given `EPOCHS` has been initialized to a number, this line is equivalent to
```python
epochs = EPOCHS
```



How about we rewrite line 208 to `EPOCHS = None` then rewrite here as
```python
epochs=EPOCHS if EPOCHS else classifier.default_training_epochs()
```

Reply:
Thanks, that's clever. Done!